google/cloud/storage/transfer_manager.py

```diff
@@ -1046,11 +1051,27 @@ def _upload_part(
         end=end,
         part_number=part_number,
         checksum=checksum,
+        headers=headers
     )
     part.upload(client._http)
     return (part_number, part.etag)
 
 
+def _headers_from_metadata(metadata):
+    """Helper function to translate object metadata into a header dictionary."""
+
+    headers = {}
+    # Handle standard writable metadata
+    for key, value in metadata.items():
+        if key in METADATA_HEADER_TRANSLATION:
```

Comment:
QQ are there any writable metadata fields that don't require translation? 

Just wanted to make sure that they are all captured in _headers_from_metadata. If not, this works!

Reply:
I believe this is the complete list according to the documentation. Note that custom metadata and some other features like encryption are not supported through this translation dictionary but handled differently.